.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!-- This form is for bug reports and feature requests ONLY!
+Also make sure that you visit our User Guide at https://kubevirt.io/user-guide/
+-->
+
+**Is this a BUG REPORT or FEATURE REQUEST?**:
+
+> Uncomment only one, leave it on its own line:
+>
+> /kind bug
+> /kind enhancement
+
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+
+**Anything else we need to know?**:
+
+**Environment**:
+- KubeVirt version (use `virtctl version`):
+- Kubernetes version (use `kubectl version`):
+- kubevirt-tekton-tasks version:
+- VM or VMI specifications:
+- Cloud provider or hardware configuration:
+- OS (e.g. from /etc/os-release):
```

Comment:
not sure if the rest (from OS) is that relevant to be in the template...

I would rather ask later in the issue for additional info if needed.